### PR TITLE
[BUG] Revise API examples for consistent tone

### LINKED ISSUE #1944: [BUG] Revise API examples for consistent tone
## Description

Some of the request/response examples in API docs contain pop culture references and snarky humor, which we should revise to be more consistent with Elastic's [voice](https://brand.elastic.co/302f66895/p/30bdc7-voice-and-tone) and [tone](https://elastic.github.io/eui/#/guidelines/writing/guidelines#be-careful-with-humor).

## Affected topics
* [Create or update an external incident](https://www.elastic.co/guide/en/security/current/cases-actions-api-execute.html#_example_requests_21) (references to James Bond, Mission: Impossible, etc.)
* Possibly other topics - part of this task will be to search the docs for any other examples
* @nastasha-solomon [pointed out](https://github.com/elastic/security-docs/pull/1987#discussion_r875880289) that several [cases API docs](https://www.elastic.co/guide/en/kibana/current/cases-api-add-comment.html) also have similar references that need revising. Cases API docs were recently moved to the Kibana repo, so https://github.com/elastic/kibana/issues/132473 will update the docs there.

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #1944. Revises API examples to remove pop culture references.

Previews:
- [Create or update an external incident](https://security-docs_1987.docs-preview.app.elstc.co/guide/en/security/master/cases-actions-api-execute.html#_example_requests_21)
- [Create rule](https://security-docs_1987.docs-preview.app.elstc.co/guide/en/security/master/rules-api-create.html)
- [Create exception container](https://security-docs_1987.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-create-container.html)
- [Create exception item](https://security-docs_1987.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-create-exception-item.html)
- [Find exception containers](https://security-docs_1987.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-find-exception-containers.html)
- [Find exception items](https://security-docs_1987.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-find-exception-items.html)
- [Get exception container](https://security-docs_1987.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-get-container.html)
- [Get exception item](https://security-docs_1987.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-get-item.html)
- [Update exception container](https://security-docs_1987.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-update-container.html)
- [Update exception item](https://security-docs_1987.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-update-item.html)
- [Create list item](https://security-docs_1987.docs-preview.app.elstc.co/guide/en/security/master/lists-api-create-list-item.html)
- [Create Timeline or Timeline template](https://security-docs_1987.docs-preview.app.elstc.co/guide/en/security/master/timeline-api-create.html)

NOTE: @nastasha-solomon [pointed out](https://github.com/elastic/security-docs/pull/1987#discussion_r875880289) that several [cases API docs](https://www.elastic.co/guide/en/kibana/current/cases-api-add-comment.html) also have similar references that need revising. Cases API docs were recently moved to the Kibana repo, so https://github.com/elastic/kibana/issues/132473 will update the docs there.

### CODE REVIEW

File: docs/cases/api/actions-api/cases-actions-api-intro.asciidoc [hunk 1]
```diff
@@ -552,12 +552,12 @@ POST api/actions/connector/7349772f-421a-4de3-b8bb-2d9b22ccee30/_execute
       "comments": [
         {
           "commentId": "dda30310-732a-11ea-a0b2-c51ea50a58e2",
-          "comment": "That is nothing - Ethan Hunt answered a targeted social media campaign promoting phishy pension schemes to IMF operatives.",
+          "comment": "Comment attached to incident",
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/cases/api/actions-api/cases-actions-api-intro.asciidoc:555]: While you can add a comment to an incident, I feel like that scenario would be better suited for an example for this API instead:
https://security-docs_1987.docs-preview.app.elstc.co/guide/en/kibana/master/cases-api-add-comment.html

File: docs/cases/api/actions-api/cases-actions-api-intro.asciidoc [hunk 2]
```diff
@@ -577,12 +577,12 @@ POST api/actions/connector/7349772f-421a-4de3-b8bb-2d9b22ccee30/_execute
...
         {
           "commentId": "8ef6d660-732f-11ea-a0b2-c51ea50a58e2",
-          "comment": "That is nothing - Ethan Hunt answered a targeted social media campaign promoting phishy pension schemes to IMF operatives.",
+          "comment": "Comment attached to incident",
         }
       ],
       "incident": {
         "externalId": "cc6ef44bdb7300106ba884da0b9619cf",
-        "short_description": "This case will self-destruct in 5 seconds"
+        "short_description": "Incident Name"
```
Comment (nastasha-solomon) [inline docs/cases/api/actions-api/cases-actions-api-intro.asciidoc:585]:
```suggestion
        "short_description": "Incident name"
```

**Review by nastasha-solomon — APPROVED** [review]
Thanks for tackling this, @joepeeples ! Left some suggestions. Feel free to take em or ignore em :)

**Review by benironside — APPROVED** [review]
Looks good!

**Pull request merged**